app/screens/global_drafts/components/tabbed_contents/tabbed_contents.test.tsx

```diff
@@ -59,15 +59,45 @@ describe('TabbedContents', () => {
             await TestHelper.wait(300); // Wait until the badge renders
         });
 
+        const scheduledTab = screen.getByTestId('scheduled_post_tab');
+        const draftTab = screen.getByTestId('draft_tab');
+
         // Check that the drafts tab is selected
-        expect(screen.getByTestId('draft_tab')).toBeTruthy();
-        expect(screen.getByTestId('drafts-content')).toBeTruthy();
+        expect(scheduledTab.props.accessibilityState).toEqual({selected: false});
+        expect(draftTab.props.accessibilityState).toEqual({selected: true});
 
         // Check that the counts are displayed correctly
         expect(screen.getByText('5')).toBeTruthy();
         expect(screen.getByText('3')).toBeTruthy();
     });
 
+    it('renders correctly with initial tab set to scheduled post', async () => {
+        const props: Parameters<typeof TabbedContents>[0] = {
+            ...defaultProps,
+            initialTab: DRAFT_SCREEN_TAB_SCHEDULED_POSTS,
+        };
+        renderWithIntlAndTheme(<TabbedContents {...props}/>);
+
+        await act(async () => {
+            await TestHelper.wait(300); // Wait until the badge renders
+        });
+
+        await act(async () => {
+            await TestHelper.wait(300);
+        });
+
+        const scheduledTab = screen.getByTestId('scheduled_post_tab');
+        const draftTab = screen.getByTestId('draft_tab');
+
+        // ✅ Check accessibilityState
```

Comment:
Nice to see clear indications that AI was used 🤪

Reply:
😎